util/emojis.py

```diff
@@ -0,0 +1,70 @@
+'''
+Utility functions for handling of emoji synchronization across guilds
+'''
+
+from io import BytesIO
+from tkinter import Image
+import discord
+import requests
+from util.log import Log
+
+async def sync_add(bot: discord.Bot, emoji: discord.Emoji):
+    for guild in bot.guilds:
+        # Check if this emoji already exists
+        guild_emojis = await guild.fetch_emojis()
+        exists = False
+        for guild_emoji in guild_emojis:
+            if guild_emoji.name == emoji.name:
+                exists = True
+                break
+        
+        # If the emoji exists, we can skip creation
+        if exists:
+            continue
+
+        #### *CAN DELETE AFTER TESTING read()*
+        # response = requests.get(emoji.url)
+        # img = Image.open(BytesIO(response.content))
+
+        # Create emoji
+        try:
+            guild.create_custom_emoji(name=emoji.name, image=emoji.read())
```

Comment:
This doesn't seem to work from my testing :(

It states that the emoji was created in the logs but it doesn't upload it to other servers. It also doesn't send a message in the bot_commands channel stating that a new emoji was uploaded and synced.

Reply:
This looks like an `await` problem. Usually uploading operations are `async`, and so [create_custom_emoji](https://docs.pycord.dev/en/master/api.html#discord.Guild.create_custom_emoji) is a coroutine.

```python
await guild.create_custom_emoji(...)
```

Reply:
Hm that is a strange one. I will take a look into it

Reply:
Ah I think it is because I made a syntax error. Overlooked that read() was a coroutine and didn't await it

Reply:
I'm not sure about read but I know that the whole `guild.create_custom_emoji(...)` line should be `await`ed, as the function to upload is a coro.